.github/labeler.yml

```diff
@@ -1,2 +1,100 @@
-Proposal:
+# Proposals
+proposal:
 - proposals/text/*
+
+# Topics
+topic:tests:
+- test/**/*
+- test/*
+- conftest.py
+topic:docker:
+- docker/*
+- docker-compose.yml
+topic:CI:
+- .github/*
+- .github/**/*
+topic:DX:
+- CONTRIBUTING.md
+- .pre-commit-config.yaml
+- .gitignore
+topic:dependencies:
+- pyproject.toml
+topic:build/distribution:
+- pyproject.toml
+topic:modeling:
+- haystack/modeling/*
+- haystack/modeling/**/*
+- test/modeling/*
+topic:reader:
+- haystack/nodes/reader/*
+- test/nodes/test_reader.py
+topic:retriever:
+- haystack/nodes/retriever/*
+- test/nodes/test_retriever.py
+topic:pipeline:
+- haystack/pipelines/*
+- test/pipelines/*
+topic:rest_api:
+- rest_api/*
+- rest_api/**/*
+topic:prompt_node:
+- haystack/nodes/prompt/*
+- test/nodes/test_prompt_node.py
+topic:file_converter:
+- haystack/nodes/file_converter/*
+- test/nodes/test_file_converter.py
+topic:crawler:
+- haystack/nodes/connector/*
+topic:preprocessing:
+- haystack/nodes/preprocessor/*
+- test/nodes/test_preprocessor.py
+topic:eval:
+- haystack/nodes/evaluator/*
+topic:images:
+- haystack/nodes/image_to_text/*
+- test/nodes/test_image_to_text.py
+topic:audio:
+- haystack/nodes/audio/*
+- test/nodes/test_audio.py
+topic:security:
+- SECURITY.md
+topic:dc-api:
+- haystack/utils/deepsetcloud.py
+topic:labelling:
+- haystack/nodes/label_generator/*
```

Comment:
Looking at how we used this label in the past, `label_generator` was only once a good fit for this label (when we used GPL): https://github.com/deepset-ai/haystack/pulls?q=is%3Apr+label%3Atopic%3Alabelling
It's so ambiguous that I'd suggest we don't use that label at all. Or rename the lable to `topic:label_generator`

Reply:
Labels can always be added manually, let's scratch it